portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,75 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+class Comment {
+  String message;
+  long timestamp;
+
+  Comment(String message, long timestamp){
+    this.message = message;
+    this.timestamp = timestamp;
+  }
+}
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    
```

Comment:
```suggestion
/** Servlet that returns some example content. */
@WebServlet("/data")
public class DataServlet extends HttpServlet {
  /** A data class to insert/get visitors' comments. */
  private class Comment {
    private String message;
    private long timestamp;

    private Comment(String message, long timestamp){
      this.message = message;
      this.timestamp = timestamp;
    }
  }    
```

Please make it a inner private class of DataServlet since it just use inside this class.